ui/dgt/src/play.ts

```diff
@@ -96,6 +96,7 @@ export default function (token: string) {
   let liveChessConnection: WebSocket; //Connection Object to LiveChess through websocket
   let isLiveChessConnected = false; //Used to track if a board there is a connection to DGT Live Chess
   let currentSerialnr = '0'; //Public property to store the current serial number of the DGT Board in case there is more than one
+  let currentBoard: CurrentBoard; //Stores active board object
```

Comment:
Adding this type is probably causing the linting error. Pretty sure it conflicts with the boards array type. Could either just edit the boards type to match CurrentBoard or just not even add the "CurrentBoard" type.